docs/USER_GUIDE.md

```diff
@@ -392,24 +392,32 @@ To grant these permissions in Lake Formation:
 4. Select a **Query Executor Type** then choose the **Database** and **Table**
    in your data catalog which describes the target data in S3. A list of columns
    will be displayed for the chosen Table.
-5. From the list, choose the column(s) the solution should use to to find items
-   in the data which should be deleted. For example, if your table has three
-   columns named **customer_id**, **description** and **created_at** and you
-   want to search for items using the **customer_id**, you should choose only
-   the **customer_id** column from this list.
-6. Enter the ARN of the role for Fargate to assume when modifying objects in S3
+5. From the Partition Keys list, select the partition key(s) that you want the
+   solution to use when generating the queries. If you select none, only one
+   query will be performed for the data mapper. If you select any or all, you'll
+   have a bigger number of smaller queries (the same query will be repeated with
+   a `WHERE` additional clause for each combination of partition values). If you
+   have a lot of small partitions, you may choose none or few partition keys
+   from the list. If instead you have very big partitions you may want to select
```

Comment:
same as above, probably want to say it may be more efficient

Reply:
Done, I added an example too.